frontend/src/services/image/image.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Http } from 'services/http/http.service';
+import { HttpMethod, ApiPath } from 'common/enums';
+
+type Constructor = {
+  http: Http;
+  apiPrefix: string;
+};
+
+class Image {
+  #http: Http;
+  #apiPrefix: string;
+
+  constructor({ http, apiPrefix }: Constructor) {
+    this.#http = http;
+    this.#apiPrefix = apiPrefix;
+  }
+
+  public addImage(image: File) {
+    const formData = new FormData();
+    formData.append('image', image);
```

Comment:
```suggestion
    formData.append('image', image);
```
We use this key for both applications. It would be better to have a constant for it